ggml/src/ggml-vulkan/vulkan-shaders/mul_mm.comp

```diff
@@ -172,7 +177,47 @@ void main() {
     const uint loadstride_b = gl_WorkGroupSize.x * LOAD_VEC_B / BK;
 
 #ifdef MUL_MAT_ID
-    uint _ne1 = 0;
+#ifdef COOPMAT
```

Comment:
It's probably a bad idea to fragment this further, but many devices without coopmat support subgroup operations, so if this is a significant improvement it may be worth enabling for non-coopmat.

Reply:
Yeah, this would also work for non-coopmat devices. But I figured it was safe to assume that coopmat support implies basic subgroup support, whereas for the non-coopmat case we'd need to compile yet another variant.